wikichron/dash/apps/monowiki/main.py

```diff
@@ -58,9 +58,11 @@ def generate_graphs(data, metrics, wikis, relative_time):
                 graphs_list[metric_idx].append(None)
         elif (category[metric_idx] == "Heatmap"):
             data[metric_idx].pop(-1)
+            graphs_list[metric_idx].append(None)
+        else:
+            graphs_list[metric_idx].append(None)
     """ Turn over data[] into plotly graphs objects and store it in graphs[] """
-
-    graphs_list = [[None for j in range(len(wikis))] for i in range(len(metrics))]
+    #graphs_list = [[None for j in range(len(wikis))] for i in range(len(metrics))]
```

Comment:
This line is incorrect in monowiki, better removing it than commenting it. Otherwise can mislead to it's relevant.

Reply:
fixed!